ui/admin/tests/acceptance/accounts/create-test.js

```diff
@@ -85,6 +85,17 @@ module('Acceptance | accounts | create', function (hooks) {
     assert.strictEqual(this.server.db.accounts.length, accountsCount + 1);
   });
 
+  test('can create a new LDAP account', async function (assert) {
+    assert.expect(1);
+    const accountsCount = this.server.db.accounts.length;
+    await visit(urls.newAccount);
+    await fillIn('[name="name"]', 'Account name');
+    await fillIn('[name="description"]', 'description');
+    await fillIn('[name="login_name"]', 'username');
+    await click('form [type="submit"]:not(:disabled)');
+    assert.strictEqual(this.server.db.accounts.length, accountsCount + 1);
```

Comment:
```suggestion
    assert.strictEqual(this.server.schema.accounts.all().models.length, accountsCount + 1);
```
there's a pattern of using `schema` over  `db` since the purpose of `schema` is to help access models.